variables.tf

```diff
@@ -168,6 +168,16 @@ variable "worker_sg_ingress_from_port" {
   default     = "1025"
 }
 
+variable "workers_addtional_policies" {
+  description = "Additional policies to be added to workers"
+  type        = "list"
+  default     = []
+}
+
+variable "workers_addtional_policies_count" {
```

Comment:
typo: `workers_addtional_policies_count` instead of `workers_additional_policies_count`

Reply:
fixed